modules/issue_tracker/jsx/columnFormatter.js

```diff
@@ -1,5 +1,43 @@
 /* exported formatColumn */
 
+class WatchingCheckbox extends React.Component {
```

Comment:
If a checkbox is a part of requirement, you could make it more generic and add as a part of `Form.js` form components. Otherwise, almost everything in Loris uses a dropdown instead. `<SelectElement>` component is  available for you in this case.